web/client/components/buttons/__tests__/ZoomToMaxExtentButton-test.jsx

```diff
@@ -194,7 +194,8 @@ describe('This test for ZoomToMaxExtentButton', () => {
             expect(actionsSpy.calls[0].arguments[2]).toNotExist();
             expect(actionsSpy.calls[0].arguments[3]).toExist();
             expect(actionsSpy.calls[0].arguments[4]).toNotExist();
-            expect(actionsSpy.calls[0].arguments[5]).toExist();
+            // the projection is null since no hook was registered
```

Comment:
What you mean for hook ? I think it's a good idea to write a test that tests this in different projections.